recipe/meta.yaml

```diff
@@ -13,6 +13,9 @@ source:
 build:
   skip: True  # [not linux]
   number: 0
+  # https://github.com/intel/cmrt/blob/master/NEWS
+  run_exports:
+    - {{ pin_subpackage('cmrt', max_pin='x.x') }}
```

Comment:
It is tough choice here. If you pin it to x.x.x, then you will need a rebuild of all the packages that depend on cmrt to get the updates for every bug-release. 

My gut instinct is to side on 'x.x' because this is a 1.0.0 release from intel, i.e. I am betting they will follow the semantic versioning guidelines of not breaking API/ABIs in revision changes.

Reply:
Yeah, that was my bet too. Lets leave it at `x.x` for now.